src/content/1.1/code/reason/snippet01.re

```diff
@@ -0,0 +1,6 @@
+module type Polymorphic_Function_F = {
```

Comment:
I don't think these module types make it easy to understand if you are not very experienced with reason. I see in Yawar's PR to the ocaml version he also simplified it.

I guess it isn't correct to say `let f: a => b;` alone, but for pedagogical reasons I feel it is better.

Reply:
I will update accordingly when his PR gets merged.